blocks/library/block/edit-panel/index.js

```diff
@@ -10,66 +11,97 @@ import { keycodes } from '@wordpress/utils';
  */
 import './style.scss';
 
+/**
```

Comment:
FWIW I don't think it's so bad in this case, because it's fairly obvious what `ESCAPE` is, but I dislike the idea of a "Module constants" docblock in general, because it normalizes the idea that we don't need to document variables we define, and can instead apply a blanket grouping to them.

Reply:
I'll leave this for now since we do this in a few places and would prefer we're somewhat consistent. It might be worth going through and auditing our usage of this pattern, since you raise a really good point.